src/components/ToggleActorBtns.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import styles from "../scenes/Game.module.css";
+
+interface ToggleActorBtnsProps {
+  onToggleActorView: () => void;
+  psychicBtn: boolean;
+  playerBtn: boolean;
+}
+
+export const ToggleActorBtns = ({
```

Comment:
Can you rename this component and file to `ActorToggle`

Reply:
Done!